server/src/utils/brute/resetBrute.ts

```diff
@@ -92,6 +97,31 @@ export const resetBrute = async ({
   );
   const randomSkill = await ServerState.getRandomSkill(prisma);
 
+  const ascendedWeapons = brute.ascendedWeapons || [];
+  const ascendedSkills = brute.ascendedSkills || [];
+  const ascendedPets = brute.ascendedPets || [];
+
+  // Add new ascent to the correspond ascended list
+  if (ascended !== undefined) {
+    if (Object.values(WeaponName).includes(ascended as WeaponName)) {
+      ascendedWeapons.push(ascended as WeaponName);
+    }
+    if (Object.values(SkillName).includes(ascended as SkillName)) {
+      ascendedSkills.push(ascended as SkillName);
+    }
+    if (Object.values(PetName).includes(ascended as PetName)) {
+      ascendedPets.push(ascended as PetName);
+    }
+
+    // Replace the destiny choice in destiny path with random stats
+    await removeDestinyChoiceFromDestinyPath(prisma, brute, ascended);
```

Comment:
This should remove the choice from the whole destiny, not only a single path

Reply:
this might be only a naming issue, currently the code removes from the whole destiny right ?

Reply:
Yes, naming issue only here